lib/utils/__tests__/isStandardSyntaxRule.test.js

```diff
@@ -4,168 +4,105 @@ const isStandardSyntaxRule = require('../isStandardSyntaxRule');
 const less = require('postcss-less');
 const postcss = require('postcss');
 
+const node = (code, parser = postcss) => parser.parse(code).first;
+const lessNode = (code) => node(code, less);
```

Comment:
[note]
- I've moved these helper functions from last to first in the file to find easily them.
- I've renamed `*rules` to `*node` because some test cases pass a non `Rule` node to the `isStandardSyntaxRule` function. E.g. `lessNode('#mixin-name;')`